docker/local-universe/Makefile

```diff
@@ -14,6 +14,12 @@ clean:
 	rm -rf certs &&							\
 	rm -f local-universe.tar.gz || 0
 
+static:
+	docker build -t universe-static -f Dockerfile.static .
+
+static-base:
+	docker build -t universe-base -f Dockerfile.static.base .
```

Comment:
Should this target replace the `base` target, since they produce the same Docker image? Even if not, our CI for building local universe will need to be updated to use `static-base`, correct?